resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DeletedKubernetesClusterChecker.scala

```diff
@@ -0,0 +1,49 @@
+package com.broadinstitute.dsp
+package resourceValidator
+
+import cats.effect.{Concurrent, Timer}
+import cats.implicits._
+import cats.mtl.ApplicativeAsk
+import io.chrisdavenport.log4cats.Logger
+import org.broadinstitute.dsde.workbench.google2.GKEModels.KubernetesClusterId
+import org.broadinstitute.dsde.workbench.model.TraceId
+
+object DeletedKubernetesClusterChecker {
```

Comment:
you'll need to hook up these checkers in https://github.com/broadinstitute/leonardo-cron-jobs/blob/master/resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ResourceValidator.scala#L42 as well